pkg/agent/qrm-plugins/network/staticpolicy/policy.go

```diff
@@ -198,34 +231,59 @@ func (p *StaticPolicy) ResourceName() string {
 	return string(apiconsts.ResourceNetBandwidth)
 }
 
+func (p *StaticPolicy) filterAvailableNICsByBandwidth(req *pluginapi.ResourceRequest) ([]machine.InterfaceInfo, error) {
```

Comment:
maybe it's better to move `exported function` together, while `none-exported function` together

Reply:
OK